scripts/install_third_party_libs.py

```diff
@@ -236,6 +236,7 @@ def main(args=None):
     setup.main(args=[])
     setup_gae.main(args=[])
     pip_dependencies = [
+        ('coverage', '4.5.4', common.OPPIA_TOOLS_DIR),
```

Comment:
Out of scope.
Any chance we can make this variable global constant? So we do not need to hardcode the directory names like this one 
https://github.com/kevinlee12/oppia/blob/b02eb7207012273025e0291a352b3e043484a64f/scripts/run_backend_tests.py#L364

Reply:
This is a good idea, I went ahead and did this.